pom.xml

```diff
@@ -132,7 +132,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials-binding</artifactId>
-            <version>1.16</version>
+            <version>1.17</version>
```

Comment:
To pick up https://github.com/jenkinsci/credentials-binding-plugin/pull/51. This seems to be the reason that, in watch mode, the test was previously failing (output was getting lost) even in situations that did not involve a missing final newline.